deployments/with-creds/vault/README.md

```diff
@@ -1,3 +1,29 @@
 # Setup
 
-1. Clone [vault-helm](https://github.com/hashicorp/vault-helm) into charts/vault-helm and checkout the version referenced in `requirements.yaml`
\ No newline at end of file
+1. ~Clone [vault-helm](https://github.com/hashicorp/vault-helm) into charts/vault-helm and checkout the version referenced in `requirements.yaml`~
+1. We have a forked version of [vault-helm](https://github.com/concourse/vault-helm). Which has something special for the postgres backend. Please keep using this version until it gets merged back to `vault-helm` chart.
+1. `vault-nci` and `vault` are alive at the same time so far. `vault` may decommission in the future after `vault-nci` is all set.
+1. How to deploy `vault-nci`
+    1. `git clone https://github.com/concourse/vault-helm` to the `charts` (hush-house/deployments/with-creds/vault/charts) directory.
+    1 `ln -s vault vault-nci` 
```

Comment:
hmmmm this seems like a transitory step that has been used during development 🤔  

do you plan to have the deployment named after `vault-nci` or `vault? I guess the former? 

(just because if the latter , then we wouldn't really need https://github.com/concourse/hush-house/pull/102 as the name of the service and namespace wouldn't change)

if you feel like the transition is good (checked via the scripts that ensure that we got the data transferred well), I'd say these steps aren't needed - we should have in this repo just the final configuration that reflect the environment 🤔

Reply:
I've manually back-upped the `/vault/data` again to make the process more safe if we only keep the `vault` instance. Anyway the `PVC` is still attached to `vault` we could switch back to the FS backend anytime.

I agree.